src/CMakeLists.txt

```diff
@@ -5,4 +5,6 @@ add_library(sqlpp-mysql
 		bind_result.cpp
 		detail/connection_handle.cpp)
 
+target_link_libraries(sqlpp-mysql PRIVATE date::date sqlpp11::sqlpp11)
```

Comment:
Both of these dependencies should be public as they are part of the public interface. 

As I wrote above, I would also not fetch date and link against it again here. Date is already a public dependency of sqlpp. Furthermore,  it's good if both libraries use the same version of the date lib.